### PR TITLE
fix(warnings): remove broad wanings suppression

### PR DESCRIPTION
**Pull request by leagris**
This shell script was applied to remove all instances of
`@SuppressWarnings("ALL")`:

```sh
grep -lr '@SuppressWarnings([[:space:]]*"ALL"[[:space:]]*)' src/ |
  xargs -l1 \
  sed -n -i.bak '/@SuppressWarnings([[:space:]]*"ALL"[[:space:]]*)/!p'
```

### CODE REVIEW

**Comment by mitchej123** [comment]
Are there some more specific warning suppressions to add in place of the all?

**Review by mitchej123 — APPROVED** [review]

**Comment by miozune** [comment]
`GT_NEI_DefaultHandler` also has `@SuppressWarnings("all")`

**Comment by leagris** [comment]
I addressed all the re-enabled warnings. Good for final review and merge.

File: src/main/java/gregtech/nei/GT_NEI_DefaultHandler.java [hunk 1]
```diff
@@ -749,38 +753,6 @@ private void drawNEIItemGrid(ItemStack[] ItemArray, int x_coord_origin, int y_co
             }
         }
 
-        // Draws a grid of fluids for NEI rendering.
-        private void drawNEIFluidGrid(FluidStack[] FluidArray, int x_coord_origin, int y_coord_origin, int x_dir_max_fluids, int y_max_dir_max_fluids, GT_Recipe Recipe, boolean is_input) {
```
Comment (mitchej123) [inline src/main/java/gregtech/nei/GT_NEI_DefaultHandler.java:753]: Do any addons use this?

Comment (mitchej123) [reply src/main/java/gregtech/nei/GT_NEI_DefaultHandler.java:753]: hah, it's private... NM

**Pull request merged**